desktop/assets/texts/es-ES/filesystem_access_dialog.ftl

```diff
@@ -0,0 +1,5 @@
+filesystem-access-dialog-title = Solicitando acceso al sistema local de archivos
+filesystem-access-dialog-message = Este contenido está tratando de leer el archivo siguiente. ¿Quieres permitirlo?
+filesystem-access-dialog-allow-remember-message = No me solicitas archivos de esta carpeta tal vez por el resto de esta sesión:
```

Comment:
I believe this might be a mistranslation - the original text is ambiguous, as "don't ask me for files ..." could also mean "don't make requests towards me to give you files ...".

Reply:
@sombraguerrero - I believe you made this contribution. Am I right? In which case, could you correct it on Crowdin please?

Reply:
The "don't ask for files again" part of the sentence is meant to say that "I don't want Ruffle to pose this question to me again", and not "Ruffle is not allowed to request files again".

Okay, maybe I'm also confused now... :face_with_spiral_eyes:

Reply:
Yeah of course I'm using one, and it's admittedly far from perfect... :face_with_peeking_eye:

Reply:
That is not automatic. My interpretation of the string was "Don't ask me" (for the duration of this session).